packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -61,6 +61,13 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
   const ONE_TINYBAR = Utils.add0xPrefix(Utils.toHex(Constants.TINYBAR_TO_WEIBAR_COEF));
   const sendRawTransaction = relay.sendRawTransaction;
 
+  const resolveEvmAddress = async (tx: any) => {
+    const fromAccountInfo = await mirrorNode.get(`/accounts/${tx.from}`);
+    const toAccountInfo = await mirrorNode.get(`/accounts/${tx.to}`);
+
+    return { from: fromAccountInfo.evm_address, to: toAccountInfo.evm_address };
```

Comment:
What if `evm_address` is null from Mirror Node?
Should this method default to use the num alias value?

Reply:
Right I didn't think about it! Pushed in the fix!